src/search.cpp

```diff
@@ -1012,7 +1012,7 @@ namespace {
       newDepth = depth - 1;
 
       // Step 13. Pruning at shallow depth (~200 Elo)
-      if (  !rootNode
+      if (  !PvNode
```

Comment:
Could you please add a UCI option to switch the logic?  The option name maybe "DontAllowLMPOnPVNodes" or so.